observability/logging/logging-6.1/log6x-configuring-lokistack-otlp-6.1.adoc

```diff
@@ -0,0 +1,151 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="log6x-configuring-lokistack-otlp-6.1"]
+= OTLP data ingestion in Loki
+include::_attributes/common-attributes.adoc[]
+:context: log6x-configuring-lokistack-otlp-6.1
+
+toc::[]
+
+Loki 3.0 introduces an API endpoint using the OpenTelemetry Protocol (OTLP), providing a new way to ingest log entries into Loki. This endpoint complements the standard Push API, which is available in Loki from its initial release. Because OTLP is a standardized format not specifically designed for Loki, it requires additional configuration on Loki's side to map OpenTelemetry's data format to Loki's data model. OTLP lacks concepts such as _stream labels_ or _structured metadata_. Instead, OTLP provides metadata about log entries as *attributes*, grouped into three categories:
```

Comment:
> Loki 3.0 introduces an API endpoint using the OpenTelemetry Protocol (OTLP), providing a new way to ingest log entries into Loki. This endpoint complements the standard Push API, which is available in Loki from its initial release.

We have never explicitly mentioned the version of Loki, this would likely be quite confusing the Openshift users as they cannot select the version of Loki we provide.

Reply:
https://issues.redhat.com/browse/OBSDOCS-1494